### ISSUE TITLE
Rough indicators for remaining installation times

### ISSUE THREAD

**Issue by ocaisa** [easybuilders/easybuild-framework#2297]
It would be nice if we included a rough time guide in easyconfigs for how long an installation may take. For beginners it would be useful in that they on't be surprised that something like a compiler takes quite some time to build. The timing could be added as part of the PR or from test reports.

**Comment by boegel** [comment]
@ocaisa I understand this can be useful, but it's quite difficult to give a realistic ballpark estimate, since it depends on many factors, like where build directories are located, e.g. slow home filesystem vs `/dev/shm` vs something in between...

**Comment by ocaisa** [comment]
Yeah, I appreciate that...but having EB just sit there for an hour with no output can make a novice believe that things are just hanging. Saying something like "This installation takes seconds/minutes/an hour/hours" might be enough to keep them happy.

**Comment by boegel** [comment]
@ocaisa Well, I guess we can always print a message like `This installation may take anywhere from seconds to minutes or hours, it depends...` ;)

Maybe the `--trace` option that will be included (under `--experimental`) in EasyBuild v3.4.0 will help to provide some more context on what `eb` is chewing on? See https://github.com/easybuilders/easybuild-framework/pull/2285.
